src/meta/processors/indexMan/CreateEdgeIndexProcessor.cpp

```diff
@@ -80,6 +74,17 @@ void CreateEdgeIndexProcessor::process(const cpp2::CreateEdgeIndexReq& req) {
             continue;
         }
 
+        if (fieldNames.size() == 0) {
+            if (item.get_fields().size() != 0) {
+                checkIter->next();
+                continue;
+            } else {
+                LOG(ERROR) << "Index " << item.get_index_name() << " has existed";
+                resp_.set_code(cpp2::ErrorCode::E_EXISTED);
+                onFinished();
+                return;
+            }
+        }
```

Comment:
I think if any index exists, there is no need to create a new index for get all edges index. just use any one of them. right?

Reply:
Good question！
First of all, I totally agree with you。

If there is any index on the tag or edge, there is no need to create a new index.
If there is no index on the tag or edge, you can create an index without fields to statis the all vertices of the tag or  the all edges of  the edgetype.

Reply:
> I think if any index exists, there is no need to create a new index for get all edges index. just use any one of them. right?

Hold on... According to the code, I don't think this is implemented. For example, I can keep creating index with no fields of different names.

Reply:
My bad, forget previous comment. I mean if index exists, no need to create one more empty index.

Reply:
Yeah，Good catch！Fixed.